main.cpp

```diff
@@ -545,13 +431,125 @@ MOVE THEM to the space below this block comment and put them in numerical order
     simply CUT and PASTE them in the space provided below:
 */
 
+Thing 1) guitar
+5 properties:
+    1) make/model (std::string)
+    2) number of strings (int)
+    3) number of pickups (int)
+    4) acoustic/electric/nylon (std::string)
+    5) active pickups yes/no (bool)
+3 things it can do:
+    1) play
+    2) feedback
+    3) smash
```

Comment:
The user does these things. The thing(object) does not. Pick different actions.
- [ ] revise